code/modules/industrial_lift/tram_lift_master.dm

```diff
@@ -172,3 +175,19 @@
 
 	travelling = new_travelling
 	SEND_SIGNAL(src, COMSIG_TRAM_SET_TRAVELLING, travelling)
+
+/**
+ * Controls the doors of the tram when it departs and arrives at stations.
+ * The tram doors are in a list of tram_doors and we apply the proc on that list.
+ */
+/datum/lift_master/tram/proc/update_tram_doors(action)
+	for(var/obj/machinery/door/window/tram_door in GLOB.tram_doors)
```

Comment:
Yeah, I was thinking that. Tram doors currently don't have an ID identifying them with what tram they're on. And in lift_master would it be faster to have one proc checking for both elevator and tram doors, or a second proc checking for tram doors?

Reply:
update_lift_doors makes z-level checks and plays a sound, which is unnessecary for a tram. I think having a different proc for a tram instead of an elevator would be cleaner, but yes checking the ID of the tram it's connected to.

Reply:
`	for(var/obj/machinery/door/poddoor/elevator_door in GLOB.machines)
		if(elevator_door.id != specific_lift_id)`
Wouldn't it be less expensive to have a var with the tram doors instead of looping through GLOB.machines each time?